app/models/post.rb

```diff
@@ -0,0 +1,14 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :likes, inverse_of: 'post', dependent: :destroy
+  has_many :comments, inverse_of: 'post', dependent: :destroy
+
+  def recent_comments
+    comments.order(created_at: :desc).limit(5)
+  end
+
+  def update_counter
+    count = author.posts_counter || 0
+    author.update_columns(posts_counter: count + 1)
+  end
```

Comment:
- [ ] [OPTIONAL] Kindly use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the update_counter method every time a new post has been saved to the database, then make the update_post_counter method private